doc/src/thermo_modify.rst

```diff
@@ -158,14 +158,20 @@ block ("yaml").  This modify option overrides the *one*, *multi*, or
 
 .. versionadded:: 4May2022
 
-The *colname* keyword can be used to change the default header keyword
-for a column or field of thermodynamic output.  The setting for *ID
-string* replaces the default text with the provided string.  *ID* can be
-a positive integer when it represents the column number counting from
-the left, a negative integer when it represents the column number from
-the right (i.e., :math:`-1` is the last column/keyword), or a thermo keyword
-(or compute, fix, property, or variable reference) and then it replaces the
-string for that specific thermo keyword.
+The *colname* keyword can be used to change the default header keyword for
+a column or field of thermodynamic output.  The column names can either be
+manually set by the user, or automatically generated for certain fixes and
+computes.  The setting for *ID string* replaces the default text with the
+provided string.  *ID* can be a positive integer when it represents the
+column number counting from the left, a negative integer when it represents
+the column number from the right (i.e., :math:`-1` is the last
+column/keyword), or a thermo keyword (or compute, fix, property, or
+variable reference) and then it replaces the string for that specific
+thermo keyword.  With a setting of *auto*, certain fixes or computes will
+generate more descriptive strings as their thermo keywords, which are
+described in the 'output' section of their documentation. Current commands
+that automatically generate descriptive thermo output strings include 'fix
+nvt', 'fix npt', 'fix nph', 'compute reduce', and 'fix bond/react'.
```

Comment:
It is also not clear to me how the 3 options (ID, auto, default) interact with each other.  Does auto or default override all previous column-specific settings for ID ?  If so, that should be emphasized.  If not, then the details should be explained. Possibly auto/default only override column settings for fixes or computes ?